dev/source/docs/sitl-native-on-windows.rst

```diff
@@ -4,9 +4,11 @@
 Setting up SITL on Windows
 ==========================
 
-SITL was originally developed on Linux, but can also be built and run *natively* Windows. It can also be run on a virtual machine (Linux) hosted on Windows, Mac OSX, or Linux.
+SITL was originally developed on Linux, but can also be built and run *natively* on Windows. It can also be run on a virtual machine (Linux) hosted on Windows, Mac OSX, or Linux or by using the Windows Subsytem for Linux (WSL).
 
-This article shows how to build and run :ref:`SITL (Software In The Loop) <sitl-simulator-software-in-the-loop>` *natively* on Windows using Cygwin. These commands were tested on Windows 10 with :ref:`MAVProxy <mavproxy:home>` and Copter-4.0.0-dev.
+Currently, the simplest way to run SITL on Windows is to build the code and simulate using WSL:  :ref:`dev:building-setup-windows10`
+
```

Comment:
I would add a `::note ` around these parts with a warning about only using cygwin for building mission planner binaries.

Reply:
going to merge and add note immediately after....